week6_role_assignment_assessment.md

```
@@ -0,0 +1,443 @@
+3. Add config block, e.g.:
+
+```yaml
+contributor_roles:
+  contributor:
+    min_merged_prs_all_time: 1
+  active_contributor:
+    min_merged_prs_in_period: 1
+    period_days: 30  # or inherit activity_period_days
+```
+
+4. Implement in **`plan_discord_roles` only**, then route `apply_discord_roles` through `DiscordPlanWriter.apply_plans` for single source of truth.
```

markdownlint-cli2 (0.22.1)

[warning] Ordered list item prefix
Expected: 1; Actual: 4; Style: 1/2/3

(MD029, ol-prefix)